app/views/needs/_need-actions.html.erb

```diff
@@ -1,26 +1,33 @@
-    <%= f.hidden_field :lock_version, :value => @need.lock_version %>
+<%= form_with(model: @need, local: true, class: "need-actions-form") do |f| %>
+    <div class="need-actions">
+      <%= f.hidden_field :lock_version, :value => @need.lock_version %>
 
-    <div class="need-actions__field" id="assigned-actions__field">
-    <%= f.label :assigned_to, 'Assigned to', class: 'need-actions__label dropdown-label' %>
-    <%= f.select :assigned_to,
-      options_for_select([['Assigned to me','assigned-to-me'], ['Unassigned', '']], (@need.assigned_to.nil?) ? '' : nil) +
-        grouped_options_for_select(@assigned_to_options, @need.assigned_to, prompt: nil),
-      {}, { class: "need-actions__select dropdown dropdown-autocomplete" } %>
-    </div>
+      <div class="need-actions__field" id="assigned-actions__field">
+      <%= f.label :assigned_to, 'Assigned to', class: 'need-actions__label dropdown-label' %>
+      <%= f.select :assigned_to,
+        options_for_select([['Assigned to me','assigned-to-me'], ['Unassigned', '']], (@need.assigned_to.nil?) ? '' : nil) +
+          grouped_options_for_select(@assigned_to_options, @need.assigned_to, prompt: nil),
+        {}, { class: "need-actions__select dropdown dropdown-autocomplete" } %>
+      </div>
 
-    <div class="need-actions__field" id="priority-actions__field">
-    <%= f.label :is_urgent, 'Priority', class: 'need-actions__label dropdown-label' %>
-    <%= f.select :is_urgent, {
-        "Urgent" => true,
-        "Normal" => false
-    }, {}, { class: @need.is_urgent ? "need-actions__select need-actions__select--red dropdown" : "need-actions__select dropdown", "data-minimum-results-for-search" => "Infinity" } %>
-    </div>
+      <div class="need-actions__field" id="priority-actions__field">
+      <%= f.label :is_urgent, 'Priority', class: 'need-actions__label dropdown-label' %>
+      <%= f.select :is_urgent, {
+          "Urgent" => true,
+          "Normal" => false
+      }, {}, { class: @need.is_urgent ? "need-actions__select need-actions__select--red dropdown" : "need-actions__select dropdown", "data-minimum-results-for-search" => "Infinity" } %>
+      </div>
 
-    <div class="need-actions__field" id="status-actions__field">
-    <%= f.label :status, 'Status', class: 'need-actions__label dropdown-label' %>
-    <%= f.select :status, need_statuses, {}, { class: "need-actions__select dropdown", "data-minimum-results-for-search" => "Infinity" } %>
-    </div>
-<% end %>
+      <div class="need-actions__field" id="status-actions__field">
+      <%= f.label :status, 'Status', class: 'need-actions__label dropdown-label' %>
+      <%= f.select :status, need_statuses, {}, { class: "need-actions__select dropdown", "data-minimum-results-for-search" => "Infinity" } %>
+      </div>   
+    </div> 
+
+    <div class="send-email__field" id="send-email__field">
+      <%= f.check_box :send_email, {class: 'bulk-action-checkbox', checked: false,  id: "send-email"}, 'true', 'false' %>
```

Comment:
should checked default to false here?

Reply:
yes it defaults to false, I think they want it to be false by default and optionally set it to "True" so they can send the email, otherwise users would have to un-tick the box every time...anyway not the best user experience

Reply:
I'm just thinking that if all the checkboxes default to false, there is no point persisting it on the model

Reply:
I need to persist it because during the assessment we save that preference but the email is not sent until assessment completion...

Reply:
I see, yeah that makes sense